deal.II-toolchain/packages/cmake.package

```diff
@@ -18,7 +18,7 @@ if [ ${PLATFORM_OSTYPE} == "linux" ]; then
     PACKING=.tar.gz
     CHECKSUM=55e30dd01698e9409b576008fd22394c
     BUILDCHAIN=ignore
-
+    INSTALL_PATH=${INSTALL_PATH}/${NAME}
```

Comment:
would you mind to move this line behind `SOURCE` as of No. 38 currently, otherwise the name is not set correctly for the else-branch (just checked this)